.buildkite/release-automation/verify-linux-wheels.sh

```diff
@@ -14,7 +14,8 @@ if [[ -z "$RAY_HASH" ]]; then
     exit 1
 fi
 
-export PATH="/root/miniconda3/bin:$PATH"
+export PATH="/usr/local/bin/miniconda3/bin:$PATH"
```

Comment:
wonder if we can make the script more agnostic with the environment by doing all init in the docker file (https://askubuntu.com/questions/849470/how-do-i-activate-a-conda-environment-in-my-bashrc). 

Otherwise this is also fine ;)

Reply:
this should be activated here.. otherwise the forge image is not very reusable..

we probably should move off conda at some point..